packages/taquito-michel-codec/src/base58.ts

```diff
@@ -91,8 +91,8 @@ export class Base58DecodingError extends Error {
  *  @category Error
  *  @description
  */
-export class InvalidHashLengthError extends Error {
-  public name = 'InvalidHashLengthError';
+export class InvalidMessageLength extends Error {
+  public name = 'InvalidMessageLength';
   constructor(public message: string) {
     super(message);
   }
```

Comment:
just realised should this be moved to an error file? as well as this may be a core error since there are multiple places where we may reject parts of hashes are are invalid

Reply:
as well as are we going to stop our convention of having `....Error` ? or should it be `InvalidMessageLengtherror`